exercises/practice/allergies/.meta/example.nim

```diff
@@ -1,24 +1,16 @@
-import sequtils
-
 type
-  Allergies* = object
-    score*: int
-
-var
-  allergiesList = [
-    "eggs",
-    "peanuts",
-    "shellfish",
-    "strawberries",
-    "tomatoes",
-    "chocolate",
-    "pollen",
-    "cats"
-  ]
-
+  Allergen* = enum
+    Eggs
+    Peanuts
+    Shellfish
+    Strawberries
+    Tomatoes
+    Chocolate
+    Pollen
+    Cats
 
-proc isAllergicTo*(allergies: Allergies, allergy: string): bool =
-  (allergies.score and 1 shl allergiesList.find(allergy)) != 0
+func isAllergicTo*(allergen: Allergen, score: int): bool =
```

Comment:
I think the arguments here should be flipped because it is asking given a score, are they allergic to an allergen